assets/js/modules/tagmanager/setup.js

```diff
@@ -436,15 +443,25 @@ class TagmanagerSetup extends Component {
 							{ __( 'Account', 'google-site-kit' ) }
 						</p>
 						<h5 className="googlesitekit-settings-module__meta-item-data">
-							{ accountID || false }
+							{ accountID || __( 'Not set', 'google-site-kit' ) }
+						</h5>
+					</div>
+
+					<div className="googlesitekit-settings-module__meta-item">
+						<p className="googlesitekit-settings-module__meta-item-type">
+							{ __( 'Web Container ID', 'google-site-kit' ) }
+						</p>
+						<h5 className="googlesitekit-settings-module__meta-item-data">
+							{ settings.containerID || __( 'Not set', 'google-site-kit' ) }
```

Comment:
Not sure about the "Not set" here, we don't do this anywhere else. Maybe just `false` for now like in most other cases?

Reply:
Isn't `false` just blank though (i.e. an empty string)? My thought was that an "empty" value would be more useful than a blank value. To your point though, if we were to change this it should probably be done for all modules at once rather than in one place. I'll update this for now but let me know if you'd like me to open an issue to discuss this.